internal/analytics/deferred.go

```diff
@@ -34,10 +34,18 @@ func deferrerFilePath(cfg Configurable) string {
 func isDeferralDayAgo(cfg Configurable) bool {
 	df := deferrerFilePath(cfg)
 	stat, err := os.Stat(df)
+	if os.IsNotExist(err) {
+		tErr := fileutils.Touch(deferrerFilePath(cfg))
+		if tErr != nil {
+			logging.Errorf("Failed to create deferrer time stamp file: %v", tErr)
+		}
```

Comment:
I don't think there's any reason we should be creating this file for the purpose of this function.